jsapp/js/components/table.es6

```diff
@@ -906,21 +906,27 @@ export class DataTable extends React.Component {
 
         {Object.keys(selected).length > 0 &&
           <ui.PopoverMenu type='bulkUpdate-menu' triggerLabel={selectedLabel} >
-            {VALIDATION_STATUSES_LIST.map((item, n) => {
-              return (
-                <bem.PopoverMenu__link
-                  onClick={this.onBulkUpdateStatus}
-                  data-value={item.value}
-                  key={n}
-                >
-                  {t('Set status: ##status##').replace('##status##', item.label)}
-                </bem.PopoverMenu__link>
-              );
-            })}
+            {this.userCan('validate_submissions', this.props.asset) &&
+              <React.Fragment>
```

Comment:
This `<React.Fragment>` isn't necessary, as `.map` returns an array of elements, which is ok for React :)